docs/modules/ROOT/nav.adoc

```diff
@@ -0,0 +1,13 @@
+* xref:ios_installation.adoc[Getting the ownCloud iOS App]
+
+* Using the iOS App
```

Comment:
Why not a link?

Reply:
Good point. Will add one.

Reply:
There is no file that would suit linking to that navigation item, and all of the sub-files are already linked.